core/modules/installer/installer.manager.inc

```diff
@@ -50,37 +50,42 @@
  * checkboxes to select which ones to upgrade.
  *
  * @param $context
- *   String representing the context from which we're trying to update.
- *   Allowed values are 'module', 'theme', and 'report'.
+ *   Deprecated since 1.18.0, after all the "Update" pages have been
+ *   consolidated into a single one, under admin/config/system/updates.
  *
  * @see installer_manager_update_form_validate()
  * @see installer_manager_update_form_submit()
  * @see installer_menu()
  * @ingroup forms
  */
-function installer_manager_update_form($form, $form_state, $context) {
-  switch ($context) {
-    case 'module':
-      backdrop_set_title('Update modules');
-      break;
-    case 'theme':
-      backdrop_set_title('Update themes');
-      break;
-    case 'layout':
-      backdrop_set_title('Update layouts');
-      break;
-    case 'report':
-      backdrop_set_title('Update projects');
-      break;
-  }
-
+function installer_manager_update_form($form, $form_state, $context = NULL) {
```

Comment:
I think you can just remove the `$context` parameter entirely. If someone passes a value anyway PHP will just ignore it. Plus I don't think a page callback like this is likely to be used by any outside modules.

Reply:
Agreed 👍  ...and done.

Reply:
Looks like `$context = NULL` is still there in this latest version?
```suggestion
function installer_manager_update_form($form, $form_state) {
```